interfaces/IICHIVault.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.7.6;
+
+interface IICHIVault {
+  function collectRewards() external;
```

Comment:
There's an implied dependency here. The ICHIVault implementation that supports this is in another repo and it's not clear from this view which repo that would be.

Reply:
I could add comments clarifying this?

Reply:
Wouldn't hurt.